src/api/app/assets/stylesheets/webui2/datatables.scss

```diff
@@ -11,3 +11,7 @@ table.table-fixed {
   width: 100% !important;
   margin-left: 0 !important;
 }
+
+.slow-loading-table {
```

Comment:
You could use bootstrap's [visibility classes](https://getbootstrap.com/docs/4.2/utilities/visibility/) instead and toogle the classes.

Reply:
@Ana06 JFYI, I added this now to your commit since you are not here today. Hope that's okay:-)